File: WebTest/ClientApp/src/app/_models/student.ts
import { Grade } from './grade';
import { Transaction } from './transaction';

export class Student {
  public studentId: number;
  public gradeId: number;
  public studentName: string;
  public yiddishName: string;
  public message: string;
  public totalPoints: number;
  public grade: Grade;
  public transactions: Transaction[];
}

File: WebTest/Models/Student.cs

using System.Collections.Generic;

namespace WebTest.Models
{
    public class Student
    {
        public int StudentId { get; set; }
        public int? GradeId { get; set; }
        public string StudentName { get; set; }
        public string YiddishName { get; set; }
        public string Message { get; set; }
        public decimal? TotalPoints { get; set; }
        public virtual Grade Grade { get; set; }
        public virtual List<Transaction> Transactions { get; set; }
    }
}
